src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -3,39 +3,62 @@ import { useHistory } from "react-router-dom";
 import _ from "lodash";
 import { DataContext } from "../../../Shared/Context/DataContext";
 import { DBentry, DBkey } from "../../../Shared/Model/Isolate.model";
-import { mockDataURL } from "../../../Shared/URLs";
+import { filterURL, mockDataURL } from "../../../Shared/URLs";
 import { QueryPageComponent } from "./QueryPage.component";
 import {
     FilterInterface,
+    FilterType,
 } from "../../../Shared/Model/Filter.model";
-import {
-    FilterContext,
-} from "../../../Shared/Context/FilterContext";
+import { FilterContext } from "../../../Shared/Context/FilterContext";
 import { TableContext } from "../../../Shared/Context/TableContext";
 import { getFilterFromPath } from "../../../Core/getFilterFromPath.service";
 import { generatePathString } from "../../../Core/generatePathString.service";
 import { getFeaturesFromPath } from "../../../Core/getTableFromPath.service";
 
+interface FilterConfigApiInterface {

```

Comment:
Consider using naming convention of our other projects. All interfaces describing data from the api should be named DTO (dataTransferObject) like FilterConfigDTO. A DTO is like a struct in higher order languages. It describes an immutable object with no functionality (no methods defined on the object).
I would also suggest to add the http mode in the name like FilterConfigResponseDTO. If you send data to the API it would be FilterConfigRequestDTO.
If you question why it is DTO and not Dto: DTO is not really a part of the name it is more a meta description and naming convention for the interface. So to recognize this on a first glance it is all upper case letters. We use this convention also for our redux actions in angular like UpdateUserSOA for StoreOperationAction.